frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/LogsTabStatus.tsx

```diff
@@ -25,10 +32,22 @@ const LogsTabStatus: React.FC<LogsTabStatusProps> = ({
       <Alert
         isInline
         variant="danger"
-        title="An error occurred while retrieving the requested logs"
-        actionLinks={<AlertActionLink onClick={refresh}>Retry</AlertActionLink>}
+        title={
+          (error?.message.includes('Failed to fetch') && error?.message + ' logs') ||
+          ((podStatus === null || (podError && podStatus?.completed)) && podError?.message)
+        }
+        actionLinks={
+          error?.message.includes('Failed to fetch') && (
+            <AlertActionLink onClick={refresh}>Retry</AlertActionLink>
```

Comment:
refresh calls refresh here but really doesnt do anything it seems? You are polling which does the refresh, but the button doesnt actually make a network call

Reply:
I agree. @yih-wang do we need a "Retry" action link here? This doesn't do anything.
<img width="607" alt="Screenshot 2024-01-11 at 3 19 42 PM" src="https://github.com/opendatahub-io/odh-dashboard/assets/22885912/23e99534-fc0d-443f-972c-e1488fd43c3c">

Reply:
@manaswinidas I didn't get the reason why we can't have a button and allow users to try to connect internet again through that.

Reply:
I think it can be in there. But because the logs would poll fairly quickly, it doesn't really serve a purpose. I also noticed that for some reason, even when refresh was called, a network call would not be made.

Reply:
Am I correct in understanding that even in the presence of a connection issue, the log viewer will continue attempting to fetch logs from the server every 2 seconds? This essentially serves the same purpose as a manual refresh button, right?

 If that's the case, I agree that we could remove the retry button and change the alert description to something like "Check your network connection. The system will recover the connection when the network is restored."

Reply:
@yih-wang 

> Am I correct in understanding that even in the presence of a connection issue, the log viewer will continue attempting to fetch logs from the server every 2 seconds? This essentially serves the same purpose as a manual refresh button, right?

Yes, log viewer attempts to fetch logs every 3 seconds. Yes, this acts as a manual refresh button.

> If that's the case, I agree that we could remove the retry button and change the alert description to something like "Check your network connection. The system will recover the connection when the network is restored."

Thanks, this works.

Reply:
Updated, thanks.